tools/apiview/parsers/rust-api-parser/clean-rust-doc-output/inputs/docs/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "docs"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+
+[lints]
+workspace = true
```

Workspaces don't work if you don't actually have a root `Cargo.toml` that declares a workspace; and, in this case, I don't recommend you do. It's overhead that often isn't worth it unless you're truly taking advantage of a monorepo like we are in the Azure/azure-sdk-for-rust repo.

---

I meant to move this clean-rust-doc-output project to the rust repository.
For now, while we are developing, I thought to have them side-by-side here for easier workflow.

Will move it there and figure out how to hook up the commands in the rust repo later, after we finalize the workflow and the parser/apiview tool.

---

How are you testing it when this won't even compile? You can use workspace references when you're not in a workspace. `cargo` will fail fast.

---

Andis there a reason to move it to the rust repo when other APIView support is here as well? If we do move it, we need to talk about where. I don't want to make it part of the main workspace. There's no need to compile this each and every time we build a workspace.

---

> How are you testing it when this won't even compile? You can use workspace references when you're not in a workspace. cargo will fail fast.

Well. I have a functional version of this in my local Rust repository. This copy is intended for easier reviews and to facilitate back-and-forth discussions and for my workflow.

> And is there a reason to move it to the rust repo when other APIView support is here as well? 

My goal is to refine the rustdoc output before integrating it with the API view tool. I need to further develop this aspect.

> If we do move it, we need to talk about where. I don't want to make it part of the main workspace. There's no need to compile this each and every time we build a workspace.

I completely agree. In that case, perhaps we could utilize a PowerShell script in that location?